java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -23,6 +23,7 @@
 #include <cudf/interop.hpp>
```

Comment:
Forgot to update the copyright year in this file.
```diff
- Copyright (c) 2019-2021, NVIDIA CORPORATION.
+ Copyright (c) 2019-2022, NVIDIA CORPORATION.
```

Reply:
Thx, Done.